.github/workflows/ci.yaml

```diff
@@ -13,9 +13,6 @@ permissions:
 jobs:
   ci:
     runs-on: ubuntu-latest
-    strategy:
-      matrix:
-        go-version: [1.22.x, 1.23.x]
```

Comment:
Since this only provides protoc plugin binaries, we should be able to keep it up to date with the latest version of go.